plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -82,190 +82,414 @@ function executeCLI($scriptName, &$args) {
+			$publicationDate = $publicationDateDao->newDataObject();
+			$publicationDate->setDateFormat('05'); // List55, YYYY
+			$publicationDate->setRole('01'); // List163, Publication Date
+			$publicationDate->setDate($data->year);
+			$publicationDate->setPublicationFormatId($publicationFormatId);
+			$publicationDateDao->insertObject($publicationDate);
+
+			// Submission File.
+			$submissionDir = sprintf($format, $press->getId(), $submissionId);
+			$fileId = $fileService->add($filePath, $submissionDir . '/' . uniqid() . '.' . $extension);
+			$mimeType = PKPString::mime_content_type($filePath);
+
+			$submissionFile = $submissionFileDao->newDataObject();
+			$submissionFile->setData('submissionId', $submissionId);
+			$submissionFile->setData('uploaderUserId', $user->getId());
+			$submissionFile->setSubmissionLocale($submission->getLocale());
+			$submissionFile->setGenreId($genre->getId());
+			$submissionFile->setFileStage(SUBMISSION_FILE_PROOF);
+			$submissionFile->setAssocType(ASSOC_TYPE_REPRESENTATION);
+			$submissionFile->setData('assocId', $publicationFormatId);
+			$submissionFile->setData('createdAt', Core::getCurrentDate());
+			$submissionFile->setDateModified(Core::getCurrentDate());
+			$submissionFile->setData('mimetype', $mimeType);
+			$submissionFile->setData('fileId', $fileId);
+			$submissionFile->setData('name', pathinfo($filePath, PATHINFO_FILENAME));
+
+			// Assume open access, no price.
+			$submissionFile->setDirectSalesPrice(0);
+			$submissionFile->setSalesType('openAccess');
+
+			$submissionFileDao->insertObject($submissionFile);
+
+			// Keywords
+			$keywordsList = [$data->locale => explode(';', $data->keywords)];
+			if (count($keywordsList[$data->locale]) > 0) {
+				/** @var $submissionKeywordDao SubmissionKeywordDAO */
+				$submissionKeywordDao = DAORegistry::getDAO('SubmissionKeywordDAO');
+				$submissionKeywordDao->insertKeywords($keywordsList, $publicationId);
 			}
+
+			//Subjects
+			$subjectsList = [$data->locale => explode(";", $data->subjects)];
+			if (count($subjectsList[$data->locale]) > 0) {
+				/** @var $submissionKeywordDao SubmissionKeywordDAO */
+				$submissionSubjectDao = DAORegistry::getDAO('SubmissionSubjectDAO');
+				$submissionSubjectDao->insertSubjects($subjectsList, $publicationId);
+			}
+
+			$coverImagelocale = [];
+			$coverImage = [];
+
+			if ($data->bookCoverImage) {
+				$sanitizedCoverImageName = str_replace([' ', '_', ':'], '-', mb_strtolower($data->bookCoverImage));
+				$sanitizedCoverImageName = PKPstring::regexp_replace("/[^a-z0-9\.\-]+/", '', $sanitizedCoverImageName);
+				$sanitizedCoverImageName = basename($sanitizedCoverImageName);
+
+				$coverImage['uploadName'] = uniqid() . '-' . $sanitizedCoverImageName;
+				$coverImage['altText'] = $bookCoverImageAltText ?? '';
+
+				$destFilePath = $publicFileManager->getContextFilesPath($press->getId()) . '/' . $coverImage['uploadName'];
+				file_put_contents($destFilePath, file_get_contents($srcFilePath));
```

Comment:
The `file_get_contents()` will load the file in memory, you can just `copy()` the file.